### PR TITLE
feature: Print the full GraphQL endpoint after a successful deploy

### PR DESCRIPTION
**Pull request by Moneyba**
## Description
Print the full GraphQL endpoint after a successful deploy

## Changes
<Changes made> 

## Checks
- [ ] Project Builds
- [x] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

File: packages/framework-provider-aws-infrastructure/src/infrastructure/stacks/application-stack.ts [hunk 1]
```diff
@@ -50,6 +50,11 @@ export class ApplicationStackBuilder {
       description: 'The base URL for all the auth endpoints and for sending GraphQL mutations and queries',
     })
 
+    new CfnOutput(stack, 'graphQL', {
```
Comment (alvaroloes) [inline packages/framework-provider-aws-infrastructure/src/infrastructure/stacks/application-stack.ts:53]: I would name this output with the same style as other URLs, `graphqlURL` for consistency (Other outputs that contain a URL are "httpURL" and "websocketURL")

**Review by alvaroloes — COMMENTED** [review]
Just left a comment

**Review by NickSeagull — APPROVED** [review]
Thank you! 🙏

**Review by javiertoledo — APPROVED** [review]
LGTM, but consider @alvaroloes's comment, I think it makes sense to keep consistency

**Pull request merged**